src/AccessControl/AuthEncoding.py

```diff
@@ -11,6 +11,8 @@
 #
 ##############################################################################
 
+from __future__ import absolute_import
+
```

Comment:
Right, I was wondering why the Python 2 tests where failing.

(The time will come when we'll be free … of Python 2.)